posts/redesigning-tracks-chicago-sloan.md

```diff
@@ -0,0 +1,37 @@
+
+### Background
+
+For the last two years, James and Gary have been investigating how the languages that a programmer uses affects their efficiency, creativity, and collaboration. Supported by [a grant from the Sloan Foundation](https://sloan.org/grant-detail/8001) they have determined that a crucial part of this research is understanding how programmers learn different languages, and how one's existing knowledge helps or hinders new learning. This is the exact question that we have been asking ourselves with regards to language tracks on Exercism. How does someone's previous experience affect the pathways they need in our tracks and the style and substance of mentoring they respond best to?
+
+James, Gary, and The Sloan Foundation have decided to invest some of their resources into enabling Exercism to focus energy into redesigning our tracks to become more focused on a language's uniquenesses and to become more personalised based on the student's existing experience.
+
+### The Project
+
+We will be blogging much more about this in the coming weeks and months, but I want to outline the main aims that we're hoping to achieve with this project, and how they will affect Exercism.
+
+**Rethinking the cores of Tracks:** Our tracks aim to teach programming languages, not programming itself. We intend to encode this into our tracks rethinking our core pathways to focus them around, specifically teaching the idioms that make a language unique. We intend to design and develop new exercises that specifically focused on one idiom each. For example, in Ruby, we might have an exercise that teaches enumeration rather than looping, or in JavaScript the concept of Prototypes or Binding.
+
+**Develop wide automated-analysis coverage of core exercises:** We want a student to be able to pass through the core of a track rapidly. In designing exercises, we will deliberately focus on how we can provide 95% automated analysis coverage to the possible solutions. This means that the interactions that mentors have will be with the 5% of students that really need their help, or on more interesting and varied side exercises. As part of this, we want to develop tools that enable our mentors to contribute to the automated analysers without having to dig into the codebase.
+
+**Offer bespoke automated mentoring and mentoring hints based on previous experience:** Our final goal is to be able to tailor the feedback that a student gets to their previous experience, either in terms of the automated responses they receive or in terms of hints we give to mentors. The feedback someone needs when learning a specific language depends heavily on their previous knowledge. Their anchor points are different, and their preexisting knowledge might be harder to unlearn, so by considering someone's previous experience, we can offer a personalised and more useful experience for students.
```

Comment:
Maybe use `"more valuable experience"` instead of `"more useful experience"`?

```suggestion
**Offer bespoke automated mentoring and mentoring hints based on previous experience:** Our final goal is to be able to tailor the feedback that a student gets to their previous experience, either in terms of the automated responses they receive or in terms of hints we give to mentors. The feedback someone needs when learning a specific language depends heavily on their previous knowledge. Their anchor points are different, and their preexisting knowledge might be harder to unlearn, so by considering someone's previous experience, we can offer a personalised and more valuable experience for students.
```